lang/lexer.nex

```diff
@@ -332,7 +332,7 @@
 			lval.str = strings.ToLower(s) // uncapitalize it
 			return CAPITALIZED_IDENTIFIER
 		}
-/[a-z][a-z0-9]*/
+/[a-z]([a-z0-9:]*[a-z0-9]+)?/
```

Comment:
Unfortunately please wait for this to land in git master from my branch...
It means you'll just have to change it manually to test the resource, sorry.